src/App.jsx

```diff
@@ -1,9 +1,10 @@
+import List from "./components/exercise5/list";
 function App() {
-
   return (
     <>
+      <List />
     </>
```

Comment:
Optional: Since you are returning only one component, I think there is no need for those tags because it's not helping in any thing. You can probably check for it and see.

Reply:
I really appreciate the feedback and I have implemented them